src/library/scss/share.scss

```diff
@@ -98,6 +98,6 @@
   }
 }
 
-.search {
+.catblocks-search {
```

Comment:
If you change this class name here, where is the belonging change of the name inside the JS Code?

Reply:
Sorry for that one - did not found it in the first place and was not sure if it is even used.
What is the next step, if everything is done and working again?

Reply:
Done. Sorry again for the inconvenience. Hope this is correct. 
I am not sure about the `.catblocks-search` class in `_blockly.scss`. (I changed it too)